front_left_foot:
  angle: 150
  multiplier: 1.0
front_left_leg:
  angle: 35
  multiplier: 1.0
front_left_shoulder:
  angle: 94
  multiplier: 1.0
front_right_foot:
  angle: 49
  multiplier: -1.0
front_right_leg:
  angle: 155
  multiplier: -1.0
front_right_shoulder:
  angle: 96
  multiplier: 1.0
rear_left_foot:
  angle: 153
  multiplier: 1.0
rear_left_leg:
  angle: 55
  multiplier: 1.0
rear_left_shoulder:
  angle: 89
  multiplier: -1.0
rear_right_foot:
  angle: 53
  multiplier: -1.0
rear_right_leg:
  angle: 155
  multiplier: -1.0
rear_right_shoulder:
  angle: 80
  multiplier: -1.0
